abdulfataiaka/pluralsight-admin
# #161447224 Implement given feedback

#### What does this PR do
- Implement given feedback from reviewer

#### Description of work done
- Complete update feature
- Add toast after delete
- Add fields placeholder

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Ensure all feedback has been implemented

#### Relevant pivotal tracker story
[161447224](https://www.pivotaltracker.com/story/show/161447224)
